charts/cloudzero-agent/docs/sizing-guide.md

```diff
@@ -0,0 +1,104 @@
+### Prometheus Agent Deployment Sizing Guide
+
+#### Introduction
+
+This guide outlines the recommended settings for deploying the Prometheus agent using the CloudZero Helm chart. It includes instructions on how to configure memory and CPU resources based on the size of your cluster.
+
+#### Baseline Memory Requirements
+
+- **Base Memory:** 512Mi
+- **Base Memory Limit:** 1024Mi
+- **Additional Memory:** 0.75Gi per 100 nodes in the cluster
+
+#### Sizing Calculation
+
+It is recommended to consider the shape and size of your prometheus cluster when setting resource memory limits for the prometheus agent. To calculate the memory requirements for your cluster, one can use the following formula:
+
+![sizing formula](./assets/sizing-formula.png)
+
+> This guide uses a simple formula based on number of nodes, however your milage may vary if you have:
+> 
+> * Very large machines, with a large number of pods
+> 
+> * High churn pods or jobs. Each pod started will trigger allocation of a time seriese based metrics cache in the agent. If it restarts, a new cache is created. The cache will resize in memory for the look back period to handle remote write failures (2 hours).
+> 
+
+
+#### Sample values.yaml Configuration
+
+Modify or create a `values.yaml` file with the following content to configure the resource limits and requests for your Prometheus agent deployment. Replace `<NUMBER_OF_NODES>` with the actual number of nodes in your cluster:
+
+```yaml
+server:
+  name: server
```

Comment:
super-nit: users wouldn't need to re-enter `name: server`. I don't think this is a big deal as all, I just wouldn't want users to think they need to maintain a file with all the sub-values of a block

Reply:
That is something I wasn't sure about. Let me remove it since it is not really needed.